README.md

```diff
@@ -161,4 +161,49 @@ python version_manager.py
 - Provides clear logging of version changes and database updates.
 
 **Note**:
-The version_manager.py script is designed to work in conjunction with the initial database population performed by initial_population.py. Ensure that you have run the initial population script before using the version manager.
\ No newline at end of file
+The version_manager.py script is designed to work in conjunction with the initial database population performed by initial_population.py. Ensure that you have run the initial population script before using the version manager.
+
+## APIs documentation 
+
+A server has been implemented based on FastAPI to allow users to interact with the database with "read only access". the API allows users to retrieve versions of a specified data model and obtain the schema URL for a particular version.
+
+To start the server, run the collowing command: 
+
+```bash
+cd api
+python -m uvicorn server:app --reload
+```
+
+### API endpoints
+
+**1. List Smart Data Model Version**
+
+This endpoint retrieves all the versions associated with a specified data model.
+ 
+````
+GET /datamodel/{name}/versions
+````
+ 
+**Parameters**
+
+- name (path parameter): The name of the data model for which you want to retrieve the versions.
+
+**Responses**
+- 200 OK: Returns a list of version numbers for the specified data model.
+- 404 Not Found: If no data model with the specified name exists in the database.
+
+
+**2. Get Schema URL by Version**
+
+This endpoint retrieves the schema URL for a specific version of a given data model.
+````
+GET /datamodel/{name}/version/{version}
+````
+
+**Parameters**
+- name (path parameter): The name of the data model.
```

Comment:
The name of the data model (Entity Type) for which you want to retrieve the versions (e.g., WasteWaterJunction).